eth/.env.example

```diff
@@ -1,7 +1,7 @@
 GENESIS_BLOCK=98
-AUCTION="0x317113D2593e3efF1FfAE0ba2fF7A61861Df7ae5"
-AUCTION_TEST="0x2b7dEe2CF60484325716A1c6A193519c8c3b19F3"
-TOKENHEZ="0x5D94e3e7aeC542aB0F9129B9a7BAdeb5B3Ca0f77"
-HERMEZ="0x8EEaea23686c319133a7cC110b840d1591d9AeE0"
-WDELAYER="0x5E0816F0f8bC560cB2B9e9C87187BeCac8c2021F"
-WDELAYER_TEST="0xc8F466fFeF9E9788fb363c2F4fBDdF2cAe477805"
+AUCTION="0x1d5c3dd2003118743d596d7db7ea07de6c90fb20"
+AUCTION_TEST="0x1d5c3dd2003118743d596d7db7ea07de6c90fb20"
+TOKENHEZ="0x55a1db90a5753e6ff50fd018d7e648d58a081486"
+HERMEZ="0xf08a226b67a8a9f99ccfcf51c50867bc18a54f53"
+WDELAYER="0xc6570883cc7e95d12bc2be6821570cb6433e3ece"
+WDELAYER_TEST="0xc6570883cc7e95d12bc2be6821570cb6433e3ece"
```

Comment:
I think these addresses are the old ones

Reply:
This is not relevant since:

- It's an example
- It's needed for some tests we don't run and are considered deprecated